### PR TITLE
Issue #9784: Prefer method reference above lambda expressions

### LINKED ISSUE #9784: Prefer Method Reference above lambda expressions
From https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/834

Replace lambda operator with method reference for CI to pass in 
https://github.com/checkstyle/checkstyle/blob/f868b02a37191ca662bb919f44353b55897793b0/src/test/java/com/puppycrawl/tools/checkstyle/xpath/AttributeNodeTest.java#L57
And 
https://github.com/checkstyle/checkstyle/blob/f868b02a37191ca662bb919f44353b55897793b0/src/test/java/com/puppycrawl/tools/checkstyle/xpath/AttributeNodeTest.java#L88

### PR DESCRIPTION
**Pull request by AkMo3**
Closes #9784

Sevntu PR: https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/834

contribution PR: https://github.com/checkstyle/contribution/pull/552

### CODE REVIEW

File: config/checkstyle_sevntu_checks.xml [hunk 1]
```diff
@@ -198,6 +198,9 @@
       <property name="ignoreEmptyReturns" value="false"/>
     </module>
     <module name="MoveVariableInsideIfCheck" />
+    <module name="com.github.sevntu.checkstyle.checks.coding.PreferMethodReferenceCheck">
```
Comment (romani) [inline config/checkstyle_sevntu_checks.xml:201]: Remove this update, it shy be done after release of sevntu

**Review by romani — CHANGES_REQUESTED** [review]
Item:

File: config/checkstyle_sevntu_checks.xml (hunk 1 above)
Comment (AkMo3) [reply config/checkstyle_sevntu_checks.xml]: Done

**Comment by AkMo3** [comment]
Travis Log
```
eval of CMD is starting

CMD=export PULL_REQUEST=9785 && ./.ci/travis/travis.sh pr-description

Commit messages from github: "Issue #9784: Prefer method reference above lambda expressio...

Issue number: #9784

PR Description grepped:"Closes #9784\r\n\r\nSevntu PR: https://github.com/sevntu-checkstyle/sevntu.chec

You are providing a PR for an Issue that is not approved yet,

please ask admins to approve your Issue first

PR validation failed.
```

**Comment by romani** [comment]
Approved, restarted

**Review by romani — APPROVED** [review]

**Pull request merged**